gaphor/SysML/blocks/property.py

```diff
@@ -22,9 +24,22 @@ def __init__(self, id=None, model=None):
         self.watch("subject.appliedStereotype.slot.definingFeature.name")
         self.watch("subject.appliedStereotype.slot.value", self.update_shapes)
         self.watch("subject[Classifier].useCase", self.update_shapes)
+        self.watch("subject[Property].aggregation", self.update_shapes)
```

Comment:
Don't all PropertyItem's have a Property as the subject? It isn't quite clear to me why I needed to explicitly specify the subject as a Property.

Reply:
It's all pre 3.x code. So there is no smartness to figure out the actual type for this subject.

BTW. Why is `subject[Classifier].useCase` watched? That's not applicable for us right? (probably an cut-n-paste error on my side :see_no_evil: )